Dockerfile.git

```
@@ -1,24 +1,23 @@
-# Build using git repo
-
-FROM alpine:3.20
-
-WORKDIR /home/testssl
-
-ARG BUILD_VERSION
-ARG ARCHIVE_URL=https://github.com/testssl/testssl.sh/archive/
-ARG URL=https://github.com/testssl/testssl.sh.git
-
-RUN test -n "${BUILD_VERSION}" \
-    && apk update \
-    && apk add --no-cache bash procps drill git coreutils libidn curl socat openssl xxd \
-    && git clone --depth 1 --branch ${BUILD_VERSION} $URL /home/testssl \
+FROM alpine:3.21 AS base-alpine
+RUN apk update \
+    && apk add --no-cache bash procps drill coreutils libidn curl socat openssl xxd \
```

`apk update` doesn't seem to serve any purpose for an image build?

```suggestion
RUN apk add --no-cache bash procps drill coreutils libidn curl socat openssl xxd \
```